distributions/Jamulus.1

```diff
@@ -227,48 +227,47 @@ left to right.
 does not provide feedback as to the current state of the Solo and Mute
 buttons so the controller must track and signal their state locally.
 .It Fl \-directoryfile Ar file
-.Pq directory server mode only
-enable server list persistence, storing to and loading from
-.Ar file
+.Pq Directory Server mode only
+remember registered Servers even if the Directory is restarted
```

Comment:
lowercase (also most of the others above here)…

Reply:
You've completely missed the point.